examples/runners/runners/web_runner.py

```diff
@@ -1,49 +1,51 @@
-import json
 import logging
-import tornado.httpserver
-import tornado.ioloop
-import tornado.options
-import tornado.web
+import uvicorn
+from fastapi import FastAPI
+from pydantic import BaseModel # pylint: disable=E0611
 from surround import Runner, RunMode
 from .stages import AssemblyState
 
+app = FastAPI() # pylint: disable=C0103
```

Comment:
Move all of the disable warnings to the top of the module so they only need to be added once. Can you make `app` all capitals to resolve the issue?

Reply:
Yeah, it does get the linting issue to disappear, but just looks a bit weird defining apps and assembler as capitals since they are not actually constants 😄

Reply:
Let's address this when we fix the Runner stuff.